[2024-08-25T09:33:58.569+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-25T09:33:58.597+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_cron_expression_v3.task1 manual__2024-08-25T09:33:56.874179+00:00 [queued]>
[2024-08-25T09:33:58.618+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_cron_expression_v3.task1 manual__2024-08-25T09:33:56.874179+00:00 [queued]>
[2024-08-25T09:33:58.620+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-25T09:33:58.653+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): task1> on 2024-08-25 09:33:56.874179+00:00
[2024-08-25T09:33:58.664+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2552) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-25T09:33:58.667+0000] {standard_task_runner.py:72} INFO - Started process 2553 to run task
[2024-08-25T09:33:58.667+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_cron_expression_v3', 'task1', 'manual__2024-08-25T09:33:56.874179+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_cron_expression.py', '--cfg-path', '/tmp/tmpda92ks7o']
[2024-08-25T09:33:58.669+0000] {standard_task_runner.py:105} INFO - Job 98: Subtask task1
[2024-08-25T09:33:58.756+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_cron_expression_v3.task1 manual__2024-08-25T09:33:56.874179+00:00 [running]> on host ae952724060e
[2024-08-25T09:33:58.929+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shreyas' AIRFLOW_CTX_DAG_ID='dag_with_cron_expression_v3' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-08-25T09:33:56.874179+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-25T09:33:56.874179+00:00'
[2024-08-25T09:33:58.931+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-25T09:33:58.954+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-25T09:33:58.956+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo dag with cron expression']
[2024-08-25T09:33:58.971+0000] {subprocess.py:86} INFO - Output:
[2024-08-25T09:33:58.973+0000] {subprocess.py:93} INFO - dag with cron expression
[2024-08-25T09:33:58.975+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-25T09:33:59.026+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-25T09:33:59.028+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_cron_expression_v3, task_id=task1, run_id=manual__2024-08-25T09:33:56.874179+00:00, execution_date=20240825T093356, start_date=20240825T093358, end_date=20240825T093359
[2024-08-25T09:33:59.085+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-25T09:33:59.112+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-25T09:33:59.115+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
